.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Arguably you would not change things like this from the original authors settings. Otherwise you will end up reformatting the entire library

Reply:
Fixed.